ucp/_libs/ucp_py.pyx

```diff
@@ -17,12 +18,18 @@ cdef extern from "ucp/api/ucp.h":
     ctypedef struct ucp_ep_h:
         pass
 
+    ctypedef struct ucp_worker_h:
+        pass
+
 cdef extern from "src/ucp_py_ucp_fxns.h":
     cdef struct ucx_context:
         int completed
     cdef struct data_buf:
         void* buf
 
+    cdef void *get_worker()
+    cdef void *get_ep_ptr(void *ep)
+
```

Comment:
@cjnolet Just one request -- can you move the above cython declarations to `ucp/_libs/ucp_py_ucp_fxns_wrapper.pyx` where all other such declarations exist?

Reply:
Poke @cjnolet it looks like there is a small request here.  Otherwise I think that we are good to go.

Reply:
I made the change last night but wanted to verify it end-to-end w/ cuML. It's been verified and changes pushed. Thanks, guys!